components/CovidMap.js

```diff
@@ -0,0 +1,702 @@
+    return (
+      <div id="map-container">
+        <div id="map" style={mapStyle} className="map"></div>
+        <div id="form" style={formStyle} className="form">
+        <form>
+          <div className="form-check">
+            <label>
+              <input type="radio" name="all" value="all" checked={this.state.selectedOption === "all"} onChange={this.handleOptionChange.bind(this)} className="form-check-input"/>
+              All Deaths
+            </label>
+          </div>
+          <div onChange={this.handleOptionChange.bind(this)} className="form-check">
+            <label>
+              <input type="radio" name="facility" value="facility" checked={this.state.selectedOption === "facility"} onChange={this.handleOptionChange.bind(this)} className="form-check-input"/>
+              By Facility
+            </label>
+          </div>
+          <div onChange={this.handleOptionChange.bind(this)} className="form-check">
+            <label>
+              <input type="radio" name="age" value="age" checked={this.state.selectedOption === "age"} onChange={this.handleOptionChange.bind(this)} className="form-check-input"/>
+              By Age
+            </label>
+          </div>
+          { 1 ? null :
+          <div onChange={this.handleOptionChange.bind(this)} className="form-check">
+            <label>
+              <input type="radio" name="ethnicity" value="ethnicity" checked={this.state.selectedOption === "ethnicity"} onChange={this.handleOptionChange.bind(this)} className="form-check-input"/>
+              By Ethnicity
+            </label>
+          </div>
+          }
+        </form>
+        </div>
+        <div id="legend" style={legendStyle} className="legend">
+          <div id="first" style={legendItemStyle} className="legendItem">
+            <div id="legend-first" style={legendTextStyle} className="legendText">{firstLegendText}</div>
+            <div style={legendIconStyle} className="icon"><span style={legendIconFirst} className="legendIcon"></span></div>
+          </div>
+          { (selectedOption == 'all') ? null:
+          <div>
+          <div id="second" style={legendItemStyle} className="legendItem">
+            <div id="legend-second" style={legendTextStyle} className="legendText" >{secondLegendText}</div>
+            <div style={legendIconStyle} className="icon"><span style={legendIconSecond} className="legendIcon"></span></div>
+          </div>
+          <div id="third" style={legendItemStyle} className="legendItem">
+            <div id="legend-third" style={legendTextStyle} className="legendText" >{thirdLegendText}</div>
+            <div style={legendIconStyle} className="icon"><span style={legendIconThird} className="legendIcon"></span></div>
+          </div>
+          </div>
+          }
+        </div>
+      </div>
+    );
+  }
+}
+
+Map.propTypes = {
+  db: PropTypes.shape({
+    sheet1: PropTypes.arrayOf(PropTypes.object)
+  })
+};
+
```

Comment:
Since we don't pass props into this component, can we dump the `propTypes`? (We could also stop importing `PropTypes` at the top of this file.

```suggestion
```

Reply:
Yes.